### PR TITLE
feat(design): use an inline menu instead of a dropdown

### PR DESCRIPTION
**Pull request by ncoden**
Previews:

preview pc

preview mobile

### CODE REVIEW

**Comment by ncoden** [comment]
@iamdey Please merge if it's ok

File: src/js/directive/layout/application-switcher.js [hunk 1]
```diff
@@ -4,8 +4,7 @@ export class LayoutApplicationSwitcherController {
     $scope.$on('$destroy', $ngRedux.connect(() => {
       let {name, route} = ApplicationHelper.getInfoForCurrentState();
```
Comment (iamdey) [inline src/js/directive/layout/application-switcher.js]: `route` can be omitted

**Comment by iamdey** [comment]
:+1:

File: src/js/directive/layout/application-switcher.js (hunk 1 above)
Comment (ncoden) [inline src/js/directive/layout/application-switcher.js]: ... You read PR in the wrong order.

First : https://github.com/CaliOpen/caliopen.web-client-ng/pull/38
Then : https://github.com/CaliOpen/caliopen.web-client-ng/pull/39

Comment (iamdey) [inline src/js/directive/layout/application-switcher.js]: I'm still right. but it's not important

Comment (iamdey) [inline src/js/directive/layout/application-switcher.js]: I'm still right. but it's not important

**Pull request merged**